components/wordPrompt.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+  useEffect(() => {
+    fetchWords();
+  }, []);
+  const changeWordHandler = () => {
+    setWords((lastWords) => {
+      if (lastWords.length <= 3) {
+        fetchWords();
```

Comment:
do you need this within `setWords`? I think we could do this check before we call `setWords`